banzai_nres/dbs.py

```diff
@@ -85,6 +146,19 @@ def get_phoenix_model_record(db_address, T_effective, log_g, metallicity, alpha)
     return model
 
 
+def get_closest_HR_phoenix_models(db_address, T_effective, luminosity, metallicity=0.0, alpha=0.0):
+    with banzai.dbs.get_session(db_address=db_address) as db_session:
+        order = [PhoenixModel.diff_T(T_effective), PhoenixModel.diff_luminosity(luminosity),
+                 PhoenixModel.diff_metallicity(metallicity), PhoenixModel.diff_alpha(alpha)]
+        model = db_session.query(PhoenixModel).order_by(*order).first()
```

Comment:
question for myself on how this works (will delete after discussing in pair coding)